drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -1003,6 +1004,26 @@ init_bridge = function(driver, device)
   local api_key = device:get_field(HueApi.APPLICATION_KEY_HEADER)
   local bridge_url = "https://" .. ip
 
+  if not api_key then
+    if not Discovery.api_keys[device_bridge_id] then
+      log.error_with({ hub_logs = true },
```

Comment:
these go to _both_ the hub and the driver logs, right?

Reply:
yes.